presto-orc/src/main/java/com/facebook/presto/orc/reader/ColumnReader.java

```diff
@@ -17,63 +17,68 @@
 import com.facebook.presto.orc.QualifyingSet;
 import com.facebook.presto.orc.stream.BooleanInputStream;
 import com.facebook.presto.orc.stream.LongInputStream;
-import com.facebook.presto.spi.block.Block;
 import com.facebook.presto.spi.type.Type;
 
 import java.io.IOException;
 import java.util.Arrays;
 
+import static java.util.Objects.requireNonNull;
+
 abstract class ColumnReader
         implements StreamReader
 {
-    QualifyingSet inputQualifyingSet;
-    QualifyingSet outputQualifyingSet;
-    Block block;
-    int outputChannel = -1;
-    Filter filter;
-    int columnIndex;
-    Type type;
-    int expectNumValues = 10000;
+    protected final int expectNumValues = 10000;
```

Comment:
Should this be a constant

Reply:
It should. At this point I still don't understand why it is defined here and not somewhere else. E.g. I'm wondering if 10K is coming from ORC format or not.

Reply:
@mbasmanova I think it comes from the row group size.